kratos/containers/geometry_container.h

```diff
@@ -128,8 +128,16 @@ class GeometryContainer
             return mGeometries.insert(pNewGeometry);
         } else if (&(*i) == pNewGeometry.get()) { // check if the pointee coincides
             return i;
-        } else {
-            KRATOS_ERROR << "Attempting to add Geometry with Id: " << pNewGeometry->Id() << ", unfortunately a (different) geometry with the same Id already exists!" << std::endl;
+        } else { // check if the connectivities coincide

```

Comment:
Can we start with capital?

Reply:
I also like to start the comments with capital font but I did it like this to keep it consistent with the rest of the file :see_no_evil:

Reply:
Pleaseeeee

Reply:
Done.